app/views/shared/_header.html.slim

```diff
@@ -1,14 +1,5 @@
-.logo-header.col-xs-12
-  a[href="#{url_for root_path}"]
-    | CASP
-  i#open_main_menu.fa.fa-bars.fa-lg.pull-left.visible-xs
-
-.text-right.user-header
-  .button-container
-    .username-logout
-      .hidden-xs
-        = link_to edit_user_registration_path, class: 'btn-inline-header', data: {placement: 'bottom', toggle: 'tooltip'}, title: 'My profile' do
-          = user_image_tag(current_user)
-          span.username = current_user.email
-      = link_to destroy_user_session_path, method: :delete, class: 'topbar btn btn-default', id: 'logout', data: {placement: 'bottom', toggle: 'tooltip'}, title: 'Logout' do
-        i.fa.fa-sign-out
+nav.navbar.navbar-default
```

Comment:
Header is already resolved in the layout: 
https://github.com/kubic-project/velum/pull/55/files#diff-1ccd3e5828c7330cdc2450c78847cd35R17

This fine should should be removed.